twake/frontend/src/app/services/languages/locale/en.js

```diff
@@ -1078,13 +1078,13 @@ export default {
   'scenes.app.workspaces.create_company.group_data.title':
     " We'll be quick — we just need some statistical information.",
   'scenes.app.workspaces.create_company.importations.title_1':
-    'Already working with digital tools? Upload or integrate your tools now!',
+    'Already working with digital tools? Import or integrate your tools now!',
   'scenes.app.workspaces.create_company.importations.title_2': 'No worries, you can do this later!',
   'scenes.app.workspaces.create_company.invitations.auto_add_inputs_info':
     'The number of fields increases when you press enter.',
   'scenes.app.workspaces.create_company.invitations.title_1':
-    'Before you finish, you can add people to your company.',
-  'scenes.app.workspaces.create_company.invitations.title_2': 'Add your team people.',
+    'Before you finish, you can add users to your company.',
+  'scenes.app.workspaces.create_company.invitations.title_2': 'Add your team users.',
```

Comment:
@BenoitTallandier for this one maybe not great?

Reply:
`Add some users` ?

Reply:
Add your team users. => Add users to your team.

Reply:
Done 👍🏻